changelog/unreleased/features/6833.md

```diff
@@ -0,0 +1,10 @@
+- Added `RoadComponent.language` value.
+- :warning: Changed `EHorizonEdgeMetadata.names` class from `RoadName` to `RoadComponent`.
+  To migrate change your code from:
+```kotlin
+val shielded = roadName.shielded
+```
+into:
+```kotlin
+val shielded = roadComponent.shield != null
+```
```

Comment:
```suggestion
- Added support for continuous EV alternatives in `NavigationRouteAlternativesObserver`.
- Fixed issues with map-matching to HOV-only roads.
- Set a limit of simultaneously running onboard route requests to avoid too many tasks blocking too much of the device's computing resources.
- Fixed an issue with Road Access Policy ignoring the setting to map-match to closed road sections, when enabled.
```
```
cc @averkhaturau @mskurydin to double-check